pipeline.go

```diff
@@ -0,0 +1,133 @@
+package pipeline
+
+import (
+	"errors"
+	"fmt"
+	"golang.org/x/net/context"
+	"google.golang.org/appengine/datastore"
+	"google.golang.org/appengine/log"
+)
+
+// Status constants
+type Status int
+
+const (
+	initialized Status = 0
+	broken             = 1
+	building           = 2
+	opened             = 3
+	resizing           = 4
+	updating           = 5
+	recreating         = 6
+	closing            = 8
+	closed             = 9
```

Comment:
Type of `initilized` is `pipeline.Status`, but `broken`, `building`, ... are `int`.
And `7` is skipped.

If these are not intended, I think these lines should be like this:

```diff
--- a/pipeline.go
+++ b/pipeline.go
@@ -12,15 +13,15 @@ import (
 type Status int
 
 const (
-       initialized Status = 0
-       broken             = 1
-       building           = 2
-       opened             = 3
-       resizing           = 4
-       updating           = 5
-       recreating         = 6
-       closing            = 8
-       closed             = 9
+       initialized Status = iota
+       broken
+       building
+       opened
+       resizing
+       updating
+       recreating
+       closing
+       closed
 )
 
 var processorFactory ProcessorFactory = &DefaultProcessorFactory{}
```